docs_user/modules/proc_adopting-the-loadbalancer-service.adoc

```diff
@@ -0,0 +1,34 @@
+[id="adopting-the-loadbalancer-service_{context}"]
+
+= Adopting the {loadbalancer_service}
+
+During the adoption process the {loadbalancer_first_ref} service
+must stay disabled in the new control plane.
+
+== Certificates
+Before running the script below the shell variables `CONTROLLER1_SSH` and
```

Comment:
maybe you should put an example of the format of those variables (like `ssh -i /path/to/key root@controller-1.xxx` or any useful tip)

Reply:
I tried to explain it a bit better now. You'll see it in my next commit. I don't want to push it now because I don't want the CI jobs to restart before they finish.

Reply:
I pushed the change in the meantime.